tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c

```diff
@@ -145,6 +168,20 @@ static void start_per_adv_set(struct bt_le_ext_adv *adv)
 	printk("done.\n");
 }
 
+static void set_per_adv_data(struct bt_le_ext_adv *adv)
+{
+	int err;
+
+	printk("Setting Periodic Advertising Data...");
+	err = bt_le_per_adv_set_data(adv, ad, ARRAY_SIZE(ad));
+	if (err) {
+		printk("Failed to set periodic advertising data: %d\n",
+		       err);
+		return;
+	}
+	printk("done.\n");
```

Comment:
```suggestion
	}
	
	printk("done.\n");
```

Reply:
The other functions in the test don't space the printk("done.\n") from the brace which is why i hadn't done it here
[1](https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c#LL106), [2](https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c#L119), [3](https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_advertiser.c#L132)...